search-app/src/cmr/search/site/data.clj

```diff
@@ -17,20 +17,29 @@
    [cmr.common-app.services.search.query-model :as query-model]
    [cmr.common-app.services.search.query-to-elastic :as q2e]
    [cmr.common-app.site.data :as common-data]
+   [cmr.common.config :as cfg :refer [defconfig]]
    [cmr.common.doi :as doi]
    [cmr.common.log :refer [debug error]]
    [cmr.common.mime-types :as mt]
    [cmr.common.util :refer [defn-timed]]
    [cmr.search.services.query-execution.granule-counts-results-feature :as gcrf]
    [cmr.search.services.query-service :as query-svc]
    [cmr.search.site.util :as util]
-   [cmr.transmit.config :as config]
+   [cmr.transmit.config :as transmit-config]
    [cmr.transmit.metadata-db :as mdb]))
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Data utility functions
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+(defconfig metadata-preview-root
+  "URL root of the metadata preview plugin"
+  {:default "https://access.sit.earthdata.nasa.gov"})
+
+(defconfig metadata-preview-version
+  "version of the metadata preview plugin"
+  {:default "0.0.25"})
+
```

Comment:
Do you think we will be changing this in code as new versions get released, or just updating the env configs?

Reply:
This will change often, so definitely via configuration.